operate/client/src/App/Processes/MigrationView/BottomPanel/index.test.tsx

```diff
@@ -6,159 +6,59 @@
+
+const {
+  requestForPayment,
+  checkPayment,
+  shipArticles,
+  shippingSubProcess,
+  confirmDelivery,
+  MessageInterrupting,
+  TimerInterrupting,
+  MessageNonInterrupting,
+  TimerNonInterrupting,
+  MessageIntermediateCatch,
+  TimerIntermediateCatch,
+} = elements;
+
+/**
+ * Returns a custom matcher function which ignores all option elements from comboboxes.
+ */
+const getMatcherFunction = (flowNodeName: string): MatcherFunction => {
+  return (content, element) => {
+    return content === flowNodeName && element?.tagName !== 'OPTION';
+  };
 };
 
 describe('MigrationView/BottomPanel', () => {
   it('should render source flow nodes', async () => {
     render(<BottomPanel />, {wrapper: Wrapper});
 
-    expect(
-      await screen.findByRole('cell', {
-        name: new RegExp(`^${requestForPayment.name}`),
-      }),
-    ).toBeInTheDocument();
-
-    expect(
-      screen.getByRole('cell', {
-        name: new RegExp(`^${checkPayment.name}`),
-      }),
-    ).toBeInTheDocument();
-
-    expect(
-      screen.getByRole('cell', {
-        name: new RegExp(`^${shipArticles.name}`),
-      }),
-    ).toBeInTheDocument();
-
-    expect(
-      screen.getByRole('cell', {
-        name: new RegExp(`^${shippingSubProcess.name}`),
-      }),
-    ).toBeInTheDocument();
-
-    expect(
-      screen.getByRole('cell', {name: new RegExp(`^${confirmDelivery.name}`)}),
-    ).toBeInTheDocument();
-
-    expect(
-      screen.getByRole('cell', {
-        name: new RegExp(`^${MessageInterrupting.name}`),
-      }),
-    ).toBeInTheDocument();
-
-    expect(
-      screen.getByRole('cell', {
-        name: new RegExp(`^${TimerInterrupting.name}`),
-      }),
-    ).toBeInTheDocument();
-
-    expect(
-      screen.getByRole('cell', {
-        name: new RegExp(`^${MessageNonInterrupting.name}`),
-      }),
-    ).toBeInTheDocument();
-
-    expect(
-      screen.getByRole('cell', {
-        name: new RegExp(`^${TimerNonInterrupting.name}`),
-      }),
-    ).toBeInTheDocument();
-
-    // expect table to have 1 header + 9 content rows
-    expect(screen.getAllByRole('row')).toHaveLength(10);
+    expect(await screen.findByText(requestForPayment.name)).toBeInTheDocument();
+    expect(screen.getByText(checkPayment.name)).toBeInTheDocument();
+    expect(screen.getByText(shipArticles.name)).toBeInTheDocument();
+    expect(screen.getByText(shippingSubProcess.name)).toBeInTheDocument();
+    expect(screen.getByText(confirmDelivery.name)).toBeInTheDocument();
+    expect(screen.getByText(MessageInterrupting.name)).toBeInTheDocument();
+    expect(screen.getByText(TimerInterrupting.name)).toBeInTheDocument();
+    expect(screen.getByText(MessageNonInterrupting.name)).toBeInTheDocument();
+    expect(screen.getByText(TimerNonInterrupting.name)).toBeInTheDocument();
+    expect(screen.getByText(MessageIntermediateCatch.name)).toBeInTheDocument();
+    expect(screen.getByText(TimerIntermediateCatch.name)).toBeInTheDocument();
```

Comment:
I changed this, because *byRole calls are pretty expensive and resulted in test timeouts. I was thinking about testing these elements partially, but came to the conclusion that I need to test every element to keep test confidence high.

Reply:
This would be faster on Vitest, haven't had issues with `byRole` hooks in there since the migration